### PR TITLE
Story documentation

### PR DESCRIPTION
**Pull request by KrystofVavra**

### CODE REVIEW

File: src/components/pages/components/story/sidePanel/index.js [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+			<p>
+				<b>The number of sections in the side panel should be the same as in the main panel.</b>
+			</p>
+			<h2>Props</h2>
+			<ComponentPropsTable>
+				<Section name={"StorySidePanel"}>
+					<Prop name="children" type="node" />
+					<Prop name="className" type="string" />
+					<Prop name="hideNavigation" type="bool" defaultValue="false">if true - hides the navigation</Prop>
+					<Prop name="navigationIcons" type="bool" defaultValue="{home: 'ri-home', case: 'ri-dot', footer: 'ri-square'}" />
```
Comment (vlach1989) [inline src/components/pages/components/story/sidePanel/index.js]: Really bool?

File: src/components/pages/components/story/sidePanel/style.scss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+.ptr-docs-story-sidePanelExample {
+	width: fit-content;
+	height: 30rem;
+	border: 0.1px solid rgba(0, 0, 0, 0.178);
```
Comment (vlach1989) [inline src/components/pages/components/story/sidePanel/style.scss]: Is the border even visible?

**Review by vlach1989 — CHANGES_REQUESTED** [review]

File: src/components/pages/components/story/sidePanel/style.scss (hunk 2 above)
Comment (KrystofVavra) [reply src/components/pages/components/story/sidePanel/style.scss]: @vlach1989 yes, it is

File: src/components/pages/components/story/sidePanel/index.js (hunk 1 above)
Comment (KrystofVavra) [reply src/components/pages/components/story/sidePanel/index.js]: oh, corrected here [e6ac630](https://github.com/gisat/app-docs/pull/43/commits/e6ac630cc4a8f2b9d577c843f82040e31ea839aa)

**Review by vlach1989 — APPROVED** [review]

**Pull request merged**